redbot/cogs/admin/admin.py

```diff
@@ -76,8 +76,8 @@ def __init__(self):
         self.config.register_global(serverlocked=False, schema_version=0)
 
         self.config.register_guild(
-            announce_channel=None,  # Integer ID
-            selfroles=[],  # List of integer ID's
+            announce_channel=None,
+            selfroles=[],  # Integer ID  # List of integer ID's
```

Comment:
This should be changed back to how it was.
```suggestion
            announce_channel=None,  # Integer ID
            selfroles=[],  # List of integer ID's
```